jenkins-design-language/src/js/components/ResultItem.jsx

```diff
@@ -109,12 +134,21 @@ export class ResultItem extends Component {
             },
         };
 
+        if (resultClean === "running" && !this.infiniteRotationRunning) {
+            requestAnimationFrame(this.infiniteLoadingTimer);
```

Comment:
Should that not better go into `componentDidMount` and validate it in `componentWillReceiveProps`?